specreduce/extract.py

```diff
@@ -425,9 +470,9 @@ def __call__(self, image=None, trace_object=None,
         norms = []
         for col_pix in range(img.shape[disp_axis]):
             # set gaussian model's mean as column's corresponding trace value
-            fit_ext_kernel.mean_0 = trace_object.trace[col_pix]
-            # NOTE: support for variable FWHMs forthcoming and would be here
 
+            # NOTE: support for variable FWHMs forthcoming and would be here
+            fit_ext_kernel.mean_0 = mean_init_guess[col_pix]
```

Comment:
The order of the code and comment were intentional here. Could you swap them back?

Reply:
My mistake!